tests/plotting/test_plot_manager.py

```diff
@@ -447,7 +447,7 @@ def raising_callback(event: PlotRefreshEvents, _: PlotRefreshResult):
     env.plot_manager.set_refresh_callback(raising_callback)
     await run_refresh_test(env.plot_manager, expect_match=False)
     # And make sure the follow-up evens aren't fired
-    assert last_even_fired is not None and last_even_fired.value == event_to_raise.value
```

Comment:
did this test pass previously? If so, I think it would be a good idea to understand why this typo wasn't caught by CI

Reply:
I see, it was consistently misspelled.

Reply:
> 
Jup, aeb4e5f.